src/glusterfs_lib/CMakeLists.txt

```diff
@@ -21,6 +21,7 @@ add_library(
         xglfs_unlink.c
         xglfs_write.c
         xglfs_fsync.c
+        xglfs_truncate.c
```

Comment:
why xglfs code is not in this PR?

Reply:
need more explanation  coz it is xglfs code are in this PR